### ISSUE TITLE
Windows: Menu bar does not behave like a native menu bar

### ISSUE THREAD

**Issue by bpasero** [atom/electron#4330]
This is probably a known issue but here are some things that do not work with the custom menu bar Electron provides on Windows:
* accessibility is bad for screen readers, as the top level menu item is not read
* you cannot use the arrow left and right keys to navigate between top level entries
* the keybinding is being read as well as the label to screen readers, which is not what happens for native menus

There are probably more issues. I wonder how much work it would be to enable the native Windows menu in Electron.

@felixrieseberg fyi, might be another topic for a hackathon :)

**Comment by felixrieseberg** [comment]
...trip to Zurich :shipit: ?

**Comment by bpasero** [comment]
;)

**Comment by zcbenz** [comment]
I'm merging this to #2504.

We were using native menu a few years ago, but at some time Chrome decided to use custom drawn menu and we had to switch to current implementation.
